demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md

```diff
@@ -0,0 +1,69 @@
+# Filter and decorate
+
+Now that we have some data, let's work with it. In this notebook, we show how to decorate and filter our data.
```

Comment:
It feels off to start off with "Now that we have some data..." and then immediately make a new table.